### ISSUE TITLE
Info: Total reads mapped in region: 0

### ISSUE THREAD

**Issue by dwaggott** [MariaNattestad/Ribbon#10]
Not sure why it doesn't like my bam files. They are pacbio aligned with ngmlr. The features are indesl from pbhoney and refflat is being used for genes.

Each chrom I select says no mapped reads.

Any ideas on how I can debug?

**Comment by MariaNattestad** [comment]
It's not a memory issue because then it would crash instead. It is either that the chromosome name doesn't match (chr prefix usually), or that there are no reads at the coordinates where the variant is. Try extending that range in the advanced options-- in the top navigation bar.

**Comment by dwaggott** [comment]
Ah sigh, the region extraction for the gene works in samtools. I'm trying the start position of the gene, I don't see how to specify a region.

This came up.
Warning: Bam files with multiple @PG header lines have been found to have issues fetching records. This bam file has 326 lines starting with @PG. If you experience issues when fetching reads from this bam file, remove all @PG lines except one. Loaded alignments from 86 reference sequences (chromosomes).

**Comment by MariaNattestad** [comment]
Does it work when you follow the suggestion from that warning?

**Comment by plurie** [comment]
Hi,
I would really like to use ribbon to visualize my structural variants, but when I load the BAM and choose a location, I get back there are no reads in this location. I mapped PacBio files with NGMLR and they have a QNAME that follows this format @<MovieName> /<ZMW_number>/<subread-start>_<subread-end>, but they do not include the chromosome. In your example you used PacBio alignments and did not have any problems, how did you get these to work? I've tried multiple aligners and the mapped BAM files always have the PACBIO BAM format as oppose to the conventional format.
